docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4062,6 +4109,26 @@ supported as follows:
 * If the `index` field is not set, the server returns the configuration for all
   indirect meters in the array identified by the unique id `meter_id`.
 
+### `MeterCounterData`
+
+The `MeterCounterData` P4Runtime message represents the per color counters
+associated with the meter configuration. Whenever a meter is updated and returns
```

Comment:
Nit: 
> "Whenever a meter is updated"

Can we say "executed" instead, to match the terminology used in  psa.p4>